### ISSUE TITLE
Adobe forces me to try a photoshop trial

### ISSUE THREAD

**Issue by Alex663366** [ignaciocastro/adobe-is-dumb#8]
I need help. I've just got this problem.
![Screenshot 2024-03-02 201017](https://github.com/ignaciocastro/adobe-is-dumb/assets/150733797/63a7cfec-ee0a-4ab4-8937-3c7965f1b30d)
Although I have used the host you created and the outside forum, but this problem is still not resolved
[host.txt](https://github.com/ignaciocastro/adobe-is-dumb/files/14469466/host.txt)
Any solution?

**Comment by ignaciocastro** [comment]
**❌ Invalid** This has nothing to do with the servers blocked by my list. This repository focuses on blocking adobe's constant verification requests, and subscriptions / alternative methods of licensing are out of scope.
